lib/react_on_rails/helper.rb

```diff
@@ -91,6 +91,36 @@ def react_component(component_name, options = {})
       end
     end
 
+    def stream_react_component(component_name, options = {})
+      options = options.merge(stream?: true)
+      result = internal_react_component(component_name, options)
```

Comment:
The result is of type [StreamRequest](https://github.com/shakacode/react_on_rails_pro/pull/407/files#diff-c1d07594e0987e090d78fe43f67ee57bd751c72c4e5cec3df2ff04a8d2606c14R58)